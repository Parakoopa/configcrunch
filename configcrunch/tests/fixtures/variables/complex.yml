base:
  level_dict:
    a:
      name: "{{ parent().more.source }}"
      more:
        a: "{{ level_helper() }}"
    b:
      name: "{{ parent().simple_helper() }}"
      more:
        a: "{{ parent().more.source|upper }}" # Jinja2 filter
  more:
    source: source
    a: "{{ level_dict.a.parent().more.source }}"
    b: "{{ level_dict.a.parent().simple_helper() }}"
    c: "{{ level_dict.a.level_helper() }}"
    d: "{{ level_dict.a.parent().level_dict.b.level_helper() }}"
