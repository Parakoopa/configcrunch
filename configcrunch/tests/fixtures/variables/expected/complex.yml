base:
  level_dict:
    a:
      name: source
      more:
        a: level
    b:
      name: simple
      more:
        a: SOURCE
  more:
    source: source
    a: source
    b: simple
    c: level
    d: level
